core/src/main/scala/cats/Foldable.scala

```diff
@@ -20,9 +20,13 @@ import simulacrum._
   def foldRight[A, B](fa: F[A], b: B)(f: (A, B) => B): B
 
   /**
-   * Left for issue #62
+   * Right associative lazy fold on `F` using the folding function 'f'.
+   *
+   * This method evaluates `b` lazily (in some cases it will not be
```

Comment:
maybe we could add a comment saying why are we using `Fold`, if I understood correctly it is to implement a *stack-safe* lazy foldRight